README.md

```diff
@@ -1,14 +1,14 @@
-# ETCExplorer 
+# ETCExplorer
 
 <b>Live Version: [etherhub.io](http://etherhub.io)</b>
 
-Follow the project progress at: [ETC Block Explorer Development](https://trello.com/b/W3ftl57z/etc-block-explorer-development) 
+Follow the project progress at: [ETC Block Explorer Development](https://github.com/ethernodeio/explorer)
 
```

Comment:
is `https://github.com/ethernodeio` going to be the new default repo for the explorer project? if so, i recommend deleting the repo from `ethereumproject` to avoid confusion.